dispute_types.go

```diff
@@ -0,0 +1,158 @@
+package paypal
```

Comment:
If you have a separate file for types, then better to move all types from dispute.go here as well.